operator/konflux.bundle.Dockerfile

```
@@ -17,33 +17,54 @@ COPY . /stackrox
 WORKDIR /stackrox/operator
 
 ARG OPERATOR_IMAGE_TAG
+RUN echo "Checking required OPERATOR_IMAGE_TAG" && [[ "${OPERATOR_IMAGE_TAG}" != "" ]]
```

nit: the `&&` suggest that we are interested in the return value of the `echo`, but that's not really the case, is it?

---

Maybe turning this around would be an improvement?
```suggestion
RUN [[ "${OPERATOR_IMAGE_TAG}" != "" ]] || echo >&2 "OPERATOR_IMAGE_TAG undefined"
```

---

> nit: the && suggest that we are interested in the return value of the echo, but that's not really the case, is it?

Yes, we don't care about `echo`-s exit code. It could be `;` instead of `&&`.

The problem of
```Dockerfile
RUN [[ "${OPERATOR_IMAGE_TAG}" != "" ]] || echo >&2 "OPERATOR_IMAGE_TAG undefined"
```
is that it does not fail the build:

```
$ cat Dockerfile 
FROM registry.access.redhat.com/ubi9/ubi:latest

ARG OPERATOR_IMAGE_TAG
RUN [[ "${OPERATOR_IMAGE_TAG}" != "" ]] || echo >&2 "OPERATOR_IMAGE_TAG undefined"

$ podman build --no-cache .
WARN[0000] missing "OPERATOR_IMAGE_TAG" build argument. Try adding "--build-arg OPERATOR_IMAGE_TAG=<VALUE>" to the command line 
STEP 1/3: FROM registry.access.redhat.com/ubi9/ubi:latest
STEP 2/3: ARG OPERATOR_IMAGE_TAG
--> 3ca4b81d2e4
STEP 3/3: RUN [[ "${OPERATOR_IMAGE_TAG}" != "" ]] || echo >&2 "OPERATOR_IMAGE_TAG undefined"
OPERATOR_IMAGE_TAG undefined
COMMIT
--> 29a7474f2b9
29a7474f2b99691214192d96d65238bb45287d260fb7663ff20f8e97726a7f20

$ echo $?
0
```

In this case we'd need curly braces, `exit`, and that all starts sounding almost as big as the original `if`.

I'd love to come up with something short and concise but don't have much more ideas.